### PR TITLE
Update status.ts

### PR DESCRIPTION
**Pull request by ctf0**
follow the theme statusbar foreground instead of fixed color

followup for https://github.com/arciisine/vscode-chronicler/pull/14

### CODE REVIEW

File: src/status.ts [hunk 1]
```diff
@@ -14,11 +14,20 @@ export class RecordingStatus {
   private item: vscode.StatusBarItem;
   timeout: NodeJS.Timer;
   counting = false;
+  mainColor
```
Comment (arciisine) [inline src/status.ts]: Please add a type declaration of `vscode.ThemeColor`.

Comment (ctf0) [reply src/status.ts]: done

**Review by arciisine — APPROVED** [review]

**Comment by arciisine** [comment]
This feature will go out with the next release of VScode, as they are introducing the concept of light/dark theme awareness, and will factor into text coloring.

**Comment by arciisine** [comment]
Actually ended up getting away from setting color and letting the theme manage completely.  Couldn't find a good way to generate contrasting colors given the current theming APIs.

The update has been released in 0.1.5

**Pull request merged**